op-guide/offline-ansible-deployment.md

```diff
@@ -0,0 +1,365 @@
+---
+title: Offline Ansible Deployment
+category: operations
+---
+
+# Offline Ansible Deployment
```

Comment:
"Offline Ansible Deployment" seems not to be a frequently used phrase. Let's change it to "Offline Deployment using Ansible"